core/sail/shacl/src/main/java/org/eclipse/rdf4j/sail/shacl/ShaclSail.java

```diff
@@ -243,13 +245,19 @@ public ShaclSail(NotifyingSail baseSail) {
 		ReferenceQueue<ShaclSail> objectReferenceQueue = new ReferenceQueue<>();
 		startMonitoring(objectReferenceQueue, new PhantomReference<>(this, objectReferenceQueue), initialized,
 				executorService);
+		this.experimentalSparqlValidation = "true"
+				.equalsIgnoreCase(System.getProperty("org.eclipse.rdf4j.sail.shacl.experimentalSparqlValidation"));
```

Comment:
The system property I choose. I prefer using a system property rather than adding a new message, since this is very temporary. That way it's easy to just remove without breaking anything for anyone.￼￼